lib/airtable/table.rb

```diff
@@ -27,6 +27,28 @@ def records(options={})
       RecordSet.new(results)
     end
 
+    # Query for records using a string formula
+    # Options: limit = 100, offset = "as345g", sort = ["Name", "asc"],
+    #          fields = [Name, Email], formula = "Count > 5", view = "Main View"
+    #
+    # select(limit: 10, sort: ["Name", "asc"], formula: "Order < 2")
+    def select(options={})
+      options['sortField'], options['sortDirection'] = options.delete(:sort) if options[:sort]
+      options['maxRecords'] = options.delete(:limit) if options[:limit]
+
+      if options[:formula]
+        raise_bad_formula_error unless options[:formula].is_a? String
+        options['filterByFormula'] = options.delete(:formula)
+      end
+
+      results = self.class.get(worksheet_url, query: options).parsed_response
+      RecordSet.new(results)
+    end
```

Comment:
agreed. without existing pagination support this seemed like the smallest acceptable contribution to add support for these type of queries. I'd be happy to add some pagination support, but would like to know that @nesquena is actively maintaining this before biting off a bigger chunk of work